src/counter.rs

```diff
@@ -63,7 +61,7 @@ impl Counter {
             v
         }
     }
-    pub fn inc(&mut self, events: usize) {
+    pub fn inc(&mut self, lvl: log::Level, events: usize) {
```

Comment:
nit: please spell out `level` fully.  No need for hard to process abbreviations like `lvl` when the full word is just another 2 letters.

Reply:
Will fix it